pkg/backup/backup.go

```diff
@@ -13,14 +13,17 @@
 
 package backup
 
-import "github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1"
+import (
+	"github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1"
+	"github.com/pingcap/tidb-operator/pkg/controller"
+)
 
 // BackupManager implements the logic for manage backup.
 type BackupManager interface {
 	// Sync	implements the logic for syncing Backup.
 	Sync(backup *v1alpha1.Backup) error
 	// UpdateCondition updates the condition for a Backup.
-	UpdateCondition(backup *v1alpha1.Backup, condition *v1alpha1.BackupCondition) error
+	UpdateCondition(backup *v1alpha1.Backup, condition *v1alpha1.BackupCondition, newStatus *controller.BackupUpdateStatus) error
```

Comment:
UpdateCondition is not just update condition now, I think we need to rename it

Reply:
ok